efficiency/bin/sdl_plot_efficiency

```diff
@@ -124,12 +124,13 @@ END
 )
 
 FULLSTRSAMPLE=$(python -c "$PYTHON_CODE")
-SAMPLE=${FULLSTRSAMPLE//"TObjString = "/}
+SAMPLE=PU200
+# SAMPLE=${FULLSTRSAMPLE//"TObjString = "/}
 
-if [[ "$SAMPLE" == *"/"* ]]; then
-    SAMPLE=${SAMPLE//\//_-_}
-    SAMPLE=${SAMPLE//./_}
-fi
+# if [[ "$SAMPLE" == *"/"* ]]; then
+#     SAMPLE=${SAMPLE//\//_-_}
+#     SAMPLE=${SAMPLE//./_}
+# fi
```

Comment:
What do these changes help with? It seems to me like this is a leftover of some check but I am not sure if there is some actual reason behind them

Reply:
Ah thanks for catching.
This is accident.